src/test/groovy/edu/ie3/datamodel/utils/validation/ValidAssetInput.groovy

```diff
@@ -0,0 +1,27 @@
+/*
+ * © 2023. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+package edu.ie3.datamodel.utils.validation
+
+import edu.ie3.datamodel.models.input.AssetInput
+
+import java.time.ZonedDateTime
+
+class ValidAssetInput extends AssetInput {
```

Comment:
I think the classes `ValidAssetInput` and `InvalidAssetInput` could be unified into one, maybe `TestAssetInput` or `DummyAssetInput`
In this instance, I think it would make the tests that they are involved in even clearer.